app/src/main/res/layout/activity_transforming.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.activity_transforming` appears to be unused
 Unused resources make applications larger and slow down builds.